### ISSUE TITLE
Add Lighthouse to build script(s) to audit website as part of build process

### ISSUE THREAD

**Issue by blairnangle** [blairnangle/blairnangle.github.io#9]
The [Lighthouse CLI tool](https://developers.google.com/web/tools/lighthouse/#cli) can be used to run automated audits of a website's performance, Progressive Web App score, accessibility and SEO. One or more of these metrics could be used as a threshold by which to fail a build.

The CLI command could be executed locally as part of a pre-commit script (explicit script or using a Git hook) and as part of a CI (e.g. GitHub Actions) build.
